client/src/components/material.jsx

```diff
@@ -100,6 +104,8 @@ const Card = Styled.div`
     margin-right: 10px;
     background: url(${props => props.url});
     z-index: 10;
```

Comment:
whats this z-index doing?

Reply:
I need things to display on top of each other on hover.